test/parallel/test-util-inspect.js

```diff
@@ -1606,17 +1613,37 @@ assert.strictEqual(util.inspect('"\'${a}'), "'\"\\'${a}'");
     }
   });
   assert.strictEqual(util.inspect(value), expected);
-  assert.strictEqual(
-    util.inspect(Object.setPrototypeOf(value, null)),
-    expected
-  );
   value.foo = 'bar';
   assert.notStrictEqual(util.inspect(value), expected);
   delete value.foo;
   value[Symbol('foo')] = 'yeah';
   assert.notStrictEqual(util.inspect(value), expected);
 });
 
+[
+  [new Set([1, 2]), '[Set: null prototype] { 1, 2 }'],
```

Comment:
Would you please also add Arrays, Dataviews and ArrayBuffers? Errors, Dates and regular expressions should be handled in a different PR as they do not show subclassing at all and that should be fixed as well.